tests/auth_test.py

```diff
@@ -109,16 +110,54 @@ def test_api_key_is_normalized(self, async_session, sync_session, client):
         self.assertEqual('returned client',
                          auth.client_from_token_file(self.json_path, 'API_KEY'))
         client.assert_called_once_with(
-                'API_KEY@AMER.OAUTHAP', _, token_metadata=_)
+                'API_KEY@AMER.OAUTHAP', _, token_metadata=_, enforce_enums=_)
         sync_session.assert_called_once_with(
             'API_KEY@AMER.OAUTHAP',
             token=self.token,
             token_endpoint=_,
             update_token=_)
 
+    @no_duplicates
+    @patch('tda.auth.Client')
+    @patch('tda.auth.OAuth2Client', new_callable=MockOAuthClient)
+    @patch('tda.auth.AsyncOAuth2Client', new_callable=MockAsyncOAuthClient)
+    def test_enforce_enums_being_disabled(self, async_session, sync_session, client):
+        self.write_token()
 
-class ClientFromAccessFunctionsTest(unittest.TestCase):
+        client.return_value = 'returned client'
 
+        self.assertEqual('returned client',
+                         auth.client_from_token_file(self.json_path, API_KEY,
+                                                     enforce_enums=False))
+        client.assert_called_once_with(API_KEY, _, token_metadata=_,
+                                       enforce_enums=False)
+        sync_session.assert_called_once_with(
```

Comment:
I believe the call to the session class is orthogonal to this test's purpose? I think we can remove it here and elsewhere, no?